apps/doji-backend/src/entities/Report.ts

```diff
@@ -0,0 +1,15 @@
+import { Entity, ManyToOne, Property } from '@mikro-orm/core'
+
+import { User } from '../entities/User'
+
+@Entity()
+export class Report {
+  @ManyToOne({ primary: true })
+  reporter: User
+
+  @ManyToOne({ primary: true })
+  expert: User
```

Comment:
not sure ว่าต้องใช้อันนี้รึเปล่า? ​https://mikro-orm.io/docs/entity-references#what-is-identifiedreference

Reply:
น่าจะไม่ใช่ เพราะต้อง check role ก่อน รวมถึงถ้าใช้แบบที่ว่ามา type จะเป็น {username : string} & Ref<User> ซึ่ง assign ค่าไม่ได้ get ก้ไม่ได้